base/applications/sndvol32/dialog.c

```diff
@@ -113,9 +113,9 @@ AddDialogControl(
         ClassName = (LPWSTR)Offset;
 
         /* adjust offset */
-        Offset += wcslen(ClassName) + 1;
+        Offset += wcslen(ClassName);
```

Comment:
I don't think this is right either. You have to skip the null terminator... not skipping it would just leave you at the null.
However, Offset is a pointer to DWORD, so you probably want `Offset = (PDWORD)((PWCHAR)Offset + wcslen(ClassName) + 1` or `Offset = (PDWORD)(ClassName + wcslen(ClassName) + 1)`